modules/ROOT/pages/creating-a-new-look-feel.adoc

```diff
@@ -1,181 +1,10 @@
-
-| skin/bootstrap/portal
-| Customizations to Bootstrap for the Portal. +
-Change these files to modify the appearance of Portal pages.
-
-| skin/bootstrap/portal/main.less
-| The entry point that identifies the files to be compiled to create the Bootstrap Look & Feel for Portal pages developed with Angular.
-
-| skin/skin.config.less
-| The main LESS file that defines the appearance. +
-Change this file to modify the appearance of Bonita Portal web `<not mobile>`. You only need to modify the other LESS files if you want to change the behaviour of the pages.
-
-| tools
-| Contains special files required for compatibility with Internet Explorer.
-
-| VERSION
-| Contains the version flag. Do not update or delete this file.
-|===
-
-== Bonita Portal Mobile default Look & Feel
-
-|===
-| Component | Description
-
-| css
-| Directory containing CSS files.
-
-| css/jquery.mobile.structure-1.2.0.min.css
-| JQuery Mobile default CSS.
-
-| css/style.css
-| JQuery Mobile default theme CSS.
-
-| img
-| Directory of images. These images are the size needed for the standard layout. +
-If you change the layout, you might need to adjust the images sizes. +
-If you want to use an image that is a different size, you might need to modify the layout.
-
-| themes
-| Directory containing CSS files overriding the JQuery files (theme for Bonita mobile).
-
-| themes/images
-| Directory of images used in the themes.
-
-| themes/bonitasoft.css
-| Theme CSS.
-
-| themes/bonitasoft.min.css
-| Minified theme CSS.
-|===
-
-== Recommendation: Form footers
-
-There are sometimes problems with the appearance of form footers after migration. If this is the case, update the `moredetails.less` file of the Look & Feel to include the following definition:
-
-[source,css]
-----
-#formframe, .forms-view, .forms-view .frame {
-  height: 100%;
-  width: 100%;
-}
-
-.forms-view .toolbar {
-  margin: 30px 0 40px 0;
-  padding: 0 35px 0 40px;
-}
-
-.forms-view .toolbar.empty {
-  margin: 0;
-  padding: 0;
-}
-
-.page_performTask .body, .page_StartProcess .body, .page_DisplayCaseForm .body {
-  padding-bottom: 0 !important;
-  margin-bottom: 0 !important;
-  display: block !important;
-  overflow: hidden;
-}
-----
-
-With this definition, the form footer is displayed instead of the Portal footer, and the form's iframe is now contained in a table row. This is recommended for easier maintenance and to avoid issues at future migrations.
+The two places that still use the old Look & Feel are the wrapped PortalJs pages and the Bonita Login page. +
+PortalJs pages take into account the theme of your application. If you want to modify their style, we strongly suggest xref:theme-development.adoc[you change the theme]. +
+For the Login page, you will still need to change the old Look & Feel in the above-mentioned folder. A solution is in the works to take into account the theme of the application.
```

Comment:
> A solution is in the works

What does this mean?

Reply:
https://www.google.com/search?q=+in+the+works&client=ubuntu&hs=Euu&channel=fs&ei=XQRTYa-_B8rClwT4oLqoDw&oq=+in+the+works&gs_lcp=Cgdnd3Mtd2l6EAMyBQgAEJECMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeMgYIABAHEB4yBggAEAcQHjIGCAAQBxAeSgQIQRgBUPY4WPY4YKo7aAFwAHgAgAGIAYgBiAGSAQMwLjGYAQCgAQHAAQE&sclient=gws-wiz&ved=0ahUKEwjv4dfqz6HzAhVK4YUKHXiQDvUQ4dUDCA0&uact=5